CoastalResilienceNetwork/GeositeFramework
# Namespace styleguide form components

The styles on form components like checkboxes and radio buttons were too
broad and ended up hiding raw html elements.  These are changed to require
classes on the surrounding markup to ensure we're targeting only form
components which are trying to make use of styleguide styles.

Connects #836 

Testing
* If it doesn't already exist, load the [Benefits Explorer](https://github.com/CoastalResilienceNetwork/benefits_explorer_prototype) plugin into your environment
* Verify the raw checkboxes are visible on the "View HydroBASINS" section

![screenshot from 2017-01-11 11 49 46](https://cloud.githubusercontent.com/assets/1014341/21857620/0e773df6-d7f4-11e6-9a0d-6e30ac52bcba.png)

* Ensure the styleguide form components are still stylized (Create Map | Save & Share)

![screenshot from 2017-01-11 11 50 28](https://cloud.githubusercontent.com/assets/1014341/21857651/273377ba-d7f4-11e6-8397-0216afb0b345.png)
